src/Analysis/Codelyzer.Analysis.CSharp/Handlers/InterfaceDeclarationHandler.cs

```diff
@@ -18,24 +17,17 @@ public class InterfaceDeclarationHandler : UstNodeHandler
             var interfaceSymbol = SemanticHelper.GetDeclaredSymbol(syntaxNode, SemanticModel, OriginalSemanticModel);
             InterfaceDeclaration.Identifier = syntaxNode.Identifier.ToString();
 
-
             if (interfaceSymbol != null)
             {
-                if (interfaceSymbol.BaseType != null)
-                {
-                    InterfaceDeclaration.BaseType = interfaceSymbol.BaseType.ToString();
-                    InterfaceDeclaration.BaseTypeOriginalDefinition = GetBaseTypOriginalDefinition(interfaceSymbol);
-                }
```

Comment:
Why did we remove these properties?

Reply:
The `BaseType` for interfaces is null, according to the document.
```
The declared base type of this type, or null. The object type, interface types, and pointer types do not have a base type. The base type of a type parameter is its effective base class.
```
https://docs.microsoft.com/en-us/dotnet/api/microsoft.codeanalysis.itypesymbol.basetype?view=roslyn-dotnet-4.2.0#microsoft-codeanalysis-itypesymbol-basetype

Reply:
We are using these values to generate the graph. Check the code graph class.

Reply:
The link doesn't say it's supposed to be null. Is this the right link?

Reply:
Yes, it's the right link. It says `The object type, interface types, and pointer types do not have a base type`.

Reply:
I'll bring these properties back.